Packs/AzureSQLManagement/Integrations/AzureSQLManagement/AzureSQLManagement.py

```diff
@@ -0,0 +1,521 @@
+                                                              is_azure_monitor_target_enabled=is_azure_monitor_target_enabled,
+                                                              is_storage_secondary_key_in_use=is_storage_secondary_key_in_use,
+                                                              queue_delay_ms=queue_delay_ms,
+                                                              retention_days=retention_days,
+                                                              storage_account_access_key=storage_account_access_key,
+                                                              storage_account_subscription_id=storage_account_subscription_id,
+                                                              storage_endpoint=storage_endpoint)
+    response_hr = copy.deepcopy(response)
+    properties = response_hr.get('properties', {})
+    if properties:
+        response_hr.update(properties)
+        del response_hr['properties']
+
+    human_readable = tableToMarkdown(name='Create Or Update Database Auditing Settings', t=response_hr,
+                                     headerTransform=pascalToSpace, removeNull=True)
+
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix='AzureSQL.DbAuditPolicy',
+        outputs_key_field='id',
+        outputs=response_hr,
+        raw_response=response
+    )
+
+
+@logger
+def azure_sql_db_threat_policy_get_command(client: Client, server_name: str, db_name: str) -> CommandResults:
+    """azure_sql_db_threat_policy_get command: Returns a threat detection policies of a database
+
+    :type client: ``Client``
+    :param client: AzureSQLManagement client to use
+
+    :return:
+        A ``CommandResults`` object that is then passed to ``return_results``,
+        that contains a scan status
+
+    :rtype: ``CommandResults``
+
+    Args:
+        server_name: server name for which we want to receive threat detection policies
+        db_name: database for which we want to receive threat detection policies
+    """
+
+    threat_list = client.azure_sql_db_threat_policy_get(server_name, db_name)
+    threat = copy.deepcopy(threat_list)
+
+    properties = threat.get('properties', {})
+    if properties:
+        threat.update(properties)
+        del threat['properties']
+
+    human_readable = tableToMarkdown(name='Database Threat Detection Policies', t=threat,
+                                     headerTransform=pascalToSpace, removeNull=True)
+
+    return CommandResults(
+        readable_output=human_readable,
+        outputs_prefix='AzureSQL.DBThreatPolicy',
+        outputs_key_field='id',
+        outputs=threat,
+        raw_response=threat_list
+    )
+
+
+@logger
+def azure_sql_db_threat_policy_create_update_command(client: Client, server_name: str, db_name: str,
+                                                     state: str, disabled_alerts: str = '',
```

Comment:
best practise is not to set mutable default values, such as empty string, but `None`, with the `Optional` type hint
for example:
```python
disabled_alerts: Optional[str] = None
```

Reply:
alredy changed this